# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/streamconv/converters/nsMultiMixedConv.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/streamconv/converters/nsMultiMixedConv.cpp
--- a/netwerk/streamconv/converters/nsMultiMixedConv.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/streamconv/converters/nsMultiMixedConv.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -165,33 +165,25 @@
 nsPartChannel::GetURI(nsIURI **aURI) { return mMultipartChannel->GetURI(aURI); }
 
 NS_IMETHODIMP
-nsPartChannel::Open(nsIInputStream **result) {
+nsPartChannel::Open(nsIInputStream **aStream) {
+  nsCOMPtr<nsIStreamListener> listener;
+  nsresult rv =
+      nsContentSecurityManager::doContentSecurityCheck(this, listener);
+  NS_ENSURE_SUCCESS(rv, rv);
+
   // This channel cannot be opened!
   return NS_ERROR_FAILURE;
 }
 
 NS_IMETHODIMP
-nsPartChannel::Open2(nsIInputStream **aStream) {
-  nsCOMPtr<nsIStreamListener> listener;
-  nsresult rv =
-      nsContentSecurityManager::doContentSecurityCheck(this, listener);
-  NS_ENSURE_SUCCESS(rv, rv);
-  return Open(aStream);
-}
-
-NS_IMETHODIMP
-nsPartChannel::AsyncOpen(nsIStreamListener *aListener, nsISupports *aContext) {
-  // This channel cannot be opened!
-  return NS_ERROR_FAILURE;
-}
-
-NS_IMETHODIMP
-nsPartChannel::AsyncOpen2(nsIStreamListener *aListener) {
+nsPartChannel::AsyncOpen(nsIStreamListener *aListener) {
   nsCOMPtr<nsIStreamListener> listener = aListener;
   nsresult rv =
       nsContentSecurityManager::doContentSecurityCheck(this, listener);
   NS_ENSURE_SUCCESS(rv, rv);
-  return AsyncOpen(listener, nullptr);
+
+  // This channel cannot be opened!
+  return NS_ERROR_FAILURE;
 }
 
 NS_IMETHODIMP